about.html

```diff
@@ -0,0 +1,120 @@
+            <ul class="second-nav-content ">
+                <li><a href="" class="links"></a>About</li>
+                <li><a href="" class="links"></a>Program </li>
+                <li><a href="" class="links"></a>Join</li>
+                <li><a href="" class="links"></a>Sponsor</li>
+                <li><a href="" class="links"></a>News</li>
+                <button class="delete"  onclick="closeMobileMenu()">
+                </button>
+                <button class="nav-button"><a href=""></a>FIFA compaign</button>
+            </ul>
+        </nav>
+        <button onclick="openMobileMenu()" class="toolbar" >
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon-menu" width="44" height="44" viewBox="0 0 24 24"
+              stroke-width="1.5" stroke="#272a32" fill="none" stroke-linecap="round" stroke-linejoin="round">
+              <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+              <line x1="4" y1="6" x2="20" y2="6" />
+              <line x1="4" y1="12" x2="20" y2="12" />
+              <line x1="4" y1="18" x2="20" y2="18" />
+            </svg>
+          </button>
+    </header>
+    <section class="about about-1">
+        <div>
+            <h1>WELCOME TO FIFA WORLD <br> CUP QATAR 2022 </h1>
+        </div>
+        <div class="headlin-description about-description">
+            <p>
+                The new FIFA is modernising football to be global, accessible and inclusive in all aspects. Not just on one or two continents, but everywhere.
+
+                Under our vision to make football truly global, we will help develop football everywhere so that there are at least 50 national teams and 50 clubs from all continents that can compete at a top competitive level.            </p>
+        </div>
+        <div class="contact">
+            <p>
+                Please contact us via Email for any further questions about FIFA &
+                QATAR world cup 2022!
+              </p>
+              <a href="#">worldcup@FIFA.com </a>
+        </div>
+    </section>
+   <section class="main-about">
+        <h2>ALL FIFA</h2>
+        <p>
+            FIFA is working on a wide range of topics from improving the women's game and developing football around the world to ensuring better football governance structures.
+        </p>
+        <div class="logo-about">
+            <img src="./images/six.webp" alt="">
+        </div>
+        <h2>FIFA Council</h2>
+        <p>
+            The Bureau of the FIFA Council decided to lift the suspension that was imposed on the Football Kenya Federation (FKF) in February 2022 in accordance with article 16 of the FIFA Statutes.
+        </p>
+
+        <div class="details-about-2">
+            <div class="right">
+                <img src="./images/seven.webp" alt="">
+            </div>
+            <div class="left">
+                <img src="./images/eight.webp" alt="">
+            </div>
+        </div>
```

Comment:
- [ ] The `background-color` for the images in this section is that of color `#ec5242`. Kindly stick to the design and make your images have the same background. Kindly implement this change for both the mobile and desktop version of your project. 👍 

#### Your Design

![image](https://user-images.githubusercontent.com/77106962/205458284-95cb9873-7c2a-4574-8cf4-d463db090ba5.png)

### Behance Template

![image](https://user-images.githubusercontent.com/77106962/205458311-748cfc3b-f6b6-4d7e-a530-bb6e92670612.png)